packages/TelegramClient-Tests.package/TCTAuthTests.class/instance/defaultTimeout.st

```diff
@@ -0,0 +1,4 @@
+accessing
+defaultTimeout
+
+	^ 10
```

Comment:
Is this used anywhere? If not we could probably remove it.
Or is it a method the testing framework implicitly uses?

Reply:
It is inherited from the Testing Framework and used to determine the timeout for the test. You can also use the pragmas (<timeout>) or send a message in setUp ("timeout:"). The former is weird and the latter (which was previously used) did not work on our machines. I suppose it must have worked at some point, but using the method seemed like the better solution to us 😄 

PS: you can check for inheritance in the browser

Reply:
Yeah I only looked into the changes on this PR and did not start my image 😅
But yeah, nice 👍